bundles/org.openhab.binding.mycroft/src/main/java/org/openhab/binding/mycroft/internal/api/dto/MessageVolumeDuck.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.mycroft.internal.api.dto;
+
+import org.openhab.binding.mycroft.internal.api.MessageType;
+
+/**
+ *
+ * @author Gwendal ROULLEAU - Initial contribution
+ */
+public class MessageVolumeDuck extends BaseMessage {
+
+    public Data data = new Data();
+    public Context context = new Context();
+
+    public MessageVolumeDuck() {
+        this.type = MessageType.mycroft_volume_duck;
+    }
+
+    public static final class Data {
```

Comment:
What is the point of having this empty class?

Reply:
It's been a while since I wrote this, but I think it is because the message produced by the binding must have a "data" and a "context" inside the json object (even if empty) to be correctly parsed by Mycroft.

Reply:
Got it, so these are DTO classes anyway. I guess its fine to define them as mycroft expects them to be then.